editor.planx.uk/src/pages/FlowEditor/lib/analytics/utils.ts

```diff
@@ -243,16 +245,12 @@ const submissionDashboard = {
 };
 
 export const getAnalyticsDashboardId = ({
-  flowStatus,
   flowSlug,
   isSubmissionService,
 }: {
-  flowStatus: FlowStatus;
   flowSlug: string;
   isSubmissionService: boolean;
 }): string | undefined => {
-  if (flowStatus === "offline") return undefined;
```

Comment:
It might be simpler just to remove this guard clause, and make no other changes....?

This would mean than any flow which meets the criteria can always access an analytics page - it just might not have any analytics if it's never been turned online. 

What do you think?